feature-libs/cart/base/core/facade/active-cart.service.ts

```diff
@@ -389,6 +389,7 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
    */
   addEntry(productCode: string, quantity: number): void {
     // TODO(#13645): Support multiple, simultaneous invocation of this function, when cart is not loaded/created
+
```

Comment:
This TODO comment can be removed (13645 was closed). (I know it is not related to bopis :) )